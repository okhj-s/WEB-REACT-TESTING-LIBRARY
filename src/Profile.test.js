import React from "react";
import { render, screen } from "@testing-library/react";
import Profile from "./Profile";

describe('<Profile />', () => {
    it('matches snapshot', () => {
        const utils = render(<Profile username="hyunjiok" name="옥현지" />);
        expect(utils.container).toMatchSnapshot();
    })
    it('shows the props corrently', () => {
        render(<Profile username="hyunjiok" name="옥현지" />);
        screen.getByText("hyunjiok");
        screen.getByText("(옥현지)");
        screen.getByText(/옥/);
    })
})