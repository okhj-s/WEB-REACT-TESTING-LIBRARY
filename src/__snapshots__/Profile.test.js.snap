// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      hyunjiok
    </b>
    <span>
      (
      옥현지
      )
    </span>
  </div>
</div>
`;
